shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-context-path/src/main/java/org/apache/shenyu/springboot/starter/plugin/context/path/ContextPathPluginConfiguration.java

```diff
@@ -21,13 +21,15 @@
 import org.apache.shenyu.plugin.base.handler.PluginDataHandler;
 import org.apache.shenyu.plugin.context.path.ContextPathPlugin;
 import org.apache.shenyu.plugin.context.path.handler.ContextPathPluginDataHandler;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 /**
  * The type Context path plugin configuration.
  */
 @Configuration
+@ConditionalOnProperty(value = {"shenyu.contextPath.enabled"}, havingValue = "true", matchIfMissing = true)
```

Comment:
all plugin add  shenyu.plugins.xxxxx.enabled

Reply:
Got it. Should I also change the existing configurations? Like `shenyu.metrics.enabled` to `shenyu.plugins.metrics.enabled`, as well as other configurations under `shenyu.metrics`. `shenyu.logging.rocketmq` is also such scenario.

Reply:
BTW, `shenyu.plugins` or `shenyu.plugin`? Just saw you mentioned `shenyu.plugin` in mail.

Reply:
1. shenyu.metrics.enable ，can not change
2. shenyu.plugins.logging-rocketmq.enabled